modules/widgets/simple-payments.php

```diff
@@ -0,0 +1,330 @@
+<?php
+/*
+Plugin Name: Simple Payments
+Description: Simple Payments button implemented as a widget.
+Version: 1.0
+Author: Automattic Inc.
+Author URI: http://automattic.com/
+License: GPLv2 or later
+*/
+
+function jetpack_register_widget_simple_payments() {
+	register_widget( 'Simple_Payments_Widget' );
+}
+add_action( 'widgets_init', 'jetpack_register_widget_simple_payments' );
+
+class Simple_Payments_Widget extends WP_Widget {
+	private static $dir              = null;
+	private static $url              = null;
+	private static $product_defaults = null;
+
+	function __construct() {
+		$widget = array(
+			'classname'   => 'simple-payments',
+			'description' => __( 'Add a simple payment button.', 'jetpack' ),
+		);
+
+		parent::__construct(
+			'Simple_Payments_Widget',
+			apply_filters( 'jetpack_widget_name', __( 'Simple Payments', 'jetpack' ) ),
+			$widget
+		);
+
+		self::$dir = trailingslashit( dirname( __FILE__ ) );
+		self::$url = plugin_dir_url( __FILE__ );
+
+		add_action( 'admin_enqueue_scripts', array( __class__, 'enqueue_admin_styles' ) );
+		add_action( 'wp_enqueue_scripts', array( __class__, 'enqueue_widget_styles' ) );
+	}
+
+	public static function enqueue_admin_styles( $hook_suffix ) {
+		if ( 'widgets.php' == $hook_suffix ) {
+			wp_enqueue_style( 'simple-payments-widget-admin', self::$url . '/simple-payments/style-admin.css', array() );
+			wp_enqueue_media();
+			wp_enqueue_script( 'simple-payments-widget-admin', self::$url . '/simple-payments/admin.js', array( 'jquery' ), false, true );
+		}
+	}
+
+	public static function enqueue_widget_styles() {
+		// TODO: This would be nice, but I don't know how this function works.
+		// if ( is_active_widget( false, false, $this->id_base, true ) ) {
+			wp_enqueue_style( 'simple-payments-widget', self::$url . '/simple-payments/style.css', array() );
```

Comment:
We concatenate such stylesheets on the frontend, by adding them here:
https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L60

Could you add yours there as well?